modules/vertx-swagger-codegen/src/main/java/com/github/phiz71/vertx/swagger/codegen/JavaVertXServerGenerator.java

```diff
@@ -131,12 +131,14 @@ public String getHelp() {
     public void processOpts() {
         super.processOpts();
 
-        if (additionalProperties.containsKey(FUTURE_GENERATION_OPTION)) {
+        if (additionalProperties.containsKey(RX_INTERFACE_OPTION)) {
             apiTemplateFiles.clear();
-            apiTemplateFiles.put("apiFuture.mustache", // the template to use
+            apiTemplateFiles.put("apiRX.mustache", // the template to use
```

Comment:
you can import templates into others. So I think it's possible to split the `service.subscribe/service.handler` into 2 small templates and then import them into a single api template.
Maybe it would be easier to do that, once the 2 PR are committed ?